### DISCUSSION TITLE
DISABLE RESIZE FEATURE ELEMENT

### DISCUSSION THREAD

**Discussion by QuangLanh** [clientIO/joint#2639] [Q&A]
### Introduction

Hello,
How can I disable the resize feature of this element?
![image](https://github.com/clientIO/joint/assets/91791283/c68097c6-cc1c-4688-bbbc-6d7f34630edc)
Please reply early, we need demo in this weekend !

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS+

**Comment by kumilingus** [comment]
It's the `ui.FreeTransform` plugin added to the element view. You have to prevent free transform from creating in the application code.

**Reply by QuangLanh** [reply]
i need disable when created element. When I first create it, I still needed this feature. But in the next step I need to disable it.

**Reply by kumilingus** [reply]
What exactly is **the next step**?

Note that you can remove the `ui.FreeTransform` instance at any time by calling `freeTransform.remove()`.
Or you can decide not to show it e.g when the user clicks an element (`element:pointerup`).
